pagopa/io-app
# feat(IT Wallet): [SIW-252] Add activation screen

## Short description
This PR adds the activation screen.

https://github.com/pagopa/io-app/assets/49342144/47454938-ba77-4814-9256-01c0f4db6541

## List of changes proposed in this pull request
- Add new screen
- Update locales
- Add ItwTos component

## How to test
Run the app with IT WALLET FF enabled. Tap on sticky component should appear the activation screen. It should also be possibile to go back tapping on "Non Ora".
